src/lasp_core.erl

```diff
@@ -466,9 +466,10 @@ bind_var(Origin, Id, Value, MetadataFun, Store) ->
                                 state_based ->
                                     {ok, WDT, Counter0, DeltaMap0};
                                 delta_based ->
+                                    Delta = lasp_type:delta(Type, state_driven, Value, Value0),
```

Comment:
- `Value0` is the local state
- `Value` is the received state

This function extracts a `Delta` from `Value` such that:
- `Value0 \join Value = Value0 \join Delta` and `Delta` is minimum.

If `Value0 = {a, b, c}` and `Value = {a, d}`, then `Delta = {d}`.